contracts/BridgeLimiter.sol

```diff
@@ -150,6 +159,7 @@ contract BridgeLimiter is IBridgeLimiter, CommitteeUpgradeable, OwnableUpgradeab
         nonReentrant
         verifyMessageAndSignatures(message, signatures, BridgeMessage.UPDATE_BRIDGE_LIMIT)
     {
+        // TODO: do we need to validate the sourceChainID here?
```

Comment:
ok so we need to verify that the "sourceChainID" is a "supportedChain" within the SuiBridge correct? Right now the `BridgeLimiter` doesn't have a way to access the `SuiBridge`'s `isChainSupported` mapping. I know the intention with including the "sourceChainID" in the `updateLimit` message is so that we can have different limits for different routes in the future. What would be the consequence for updating the limit of an unsupported chain ID?

Reply:
there is no superficial harm but unintentional changes like these easily lead to undesired bugs down the road.

what about moving the pub interface of update limit/asset price to SuiBridge, and then expose onlyOwner functions in Limiter?